sources/packages/backend/apps/api/src/services/institution/institution.service.model.ts

```diff
@@ -34,6 +35,7 @@ export interface InstitutionFormModel {
   primaryEmail: string;
   website: string;
   regulatingBody: string;
+  otherRegulatingBody: string;
```

Comment:
`otherRegulatingBody?: string;` as its optional

Reply:
Sounds good. Will do the change.